### DISCUSSION TITLE
Is there any limit or efficiency to how many fields a remote harvester can harvest

### DISCUSSION THREAD

**Discussion by wanming851** [Chia-Network/chia-blockchain#6699] [Support]
Is there any limit or efficiency to how many fields a remote harvester can harvest

**Comment by loppefaaret** [comment]
people are running thousand of plots on a single harvester

**Reply by wanming851** [reply]
Is there a ceiling

**Reply by loppefaaret** [reply]
ofcourse, but that ceiling is dependant on your system

**Comment by BrandtH22** [comment]
@wanming851 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
